platform/platform.go

```diff
@@ -24,17 +24,32 @@ type Platform interface {
 	Regrade([]RegradeSpec) error
 }
 
+type RemotePlatform interface {
+	Platform
+	Ping() error
```

Comment:
Hmm, yeah I guess so; RPC would then call that Ping, rather than implementing it inside the mechanism.